spec/validation/auth/authSpec.coffee

```diff
@@ -1,6 +1,6 @@
 auth = require('../../../lib/validation/validate').auth
 
-super_admin         = {name: 'admin'}
+system_user         = {name: 'admin'}
 team_admin          = {name: 'etkdg394hpmujn', roles: ['kratos|enabled']}
```

Comment:
Is the `resource | status` convention something that was chosen in development? Is there a better way to do this or does this prevent additional parsing down the line?